### DISCUSSION TITLE
Feature Suggestion: Support enterprise policies/managed storage.

### DISCUSSION THREAD

**Discussion by AJolly** [furybee/chrome-tab-modifier#352] [Ideas]
Great to see this being updated again!
I'll have to work on backporting my own feature upgrades I added to my own local fork.

What do you think about adding managed storage support ? (That way we can prestore rules) 
 https://developer.chrome.com/docs/extensions/reference/manifest/storage

Right now I just ended up hardcoding a bunch of rules locally.

**Comment by sylouuu** [comment]
Hi @AJolly 

You can submit a PR with changes if you think it can be useful for other TM users.

You can import/export rules easily (fixed in `1.0.10` btw).

Bests
